bootstrap/postgres/configs.go

```diff
@@ -451,7 +464,36 @@ func (cr configRepository) RemoveChannel(ctx context.Context, id string) error {
 	return nil
 }
 
+func (cr configRepository) ConnectThing(ctx context.Context, channelID, thingID string) error {
+	if thingID == "" || channelID == "" {
+		return repoerr.ErrMalformedEntity
+	}
+
+	qCheck := `SELECT EXISTS(SELECT 1 FROM configs WHERE magistrala_thing = $1), state FROM configs WHERE magistrala_thing = $1`
```

Comment:
Is it necessary to do 2 db calls i.e. select then update?

Why not just updated if it failed we return NotFound it the ids aren't found